Rakefile

```diff
@@ -318,7 +363,7 @@ def create_anonymous_user
       anonymous_login = config['login']
       anonymous_password = config['password']
 
-      if anonymous_login and anonymous_password and not ScalarmUser.find_by_login(anonymous_login)
+      if anonymous_login and anonymous_password and not ScalarmUser.where(login: anonymous_login).first
```

Comment:
@dkrol - pierwszy powód może być taki, że z commitem wchodzi niepotrzebna zmiana, która potem będzie przypisana do tego commita przy okazji annotate (git blame).
Co do drugiego powodu, są to moje przypuszczenia: używałem ``find_by_*``, dlatego, że przeimplementowując tą metodę dodałem tam: ``{limit: 1}``, która zapewnia szybsze wyszukiwanie. Nie wiem dokładnie jak działa użycie ``where(...).first`` bez tego warunku. Jednym słowem - ``find_by_*`` może być wydajniejsze lub tak samo wydajne.